DigitalLearningSolutions.Web.AutomatedUiTests/AccessibilityTests/BasicAuthenticatedAccessibilityTests.cs

```diff
@@ -52,6 +52,7 @@ public class BasicAuthenticatedAccessibilityTests : AccessibilityTestsBase,
         [InlineData("/TrackingSystem/Delegates/Groups/Add", "Add new delegate group")]
         [InlineData("/TrackingSystem/Delegates/3/View", "xxxx xxxxxx")]
         [InlineData("/TrackingSystem/Delegates/3/SetPassword", "Set delegate user password")]
+        [InlineData("/TrackingSystem/Delegates/3/View/100/Remove", "Remove delegate enrolment from Entry Level - Win XP, Office 2003/07 OLD - Standard")]

```

Comment:
Formatting? I'm surprised this hasn't been broken across new lines like some of the above cases.